Added path /usr/lib/bsnes/ to search path so that bsnes can find it's helper libraries, such as snesreader, there.
--- a/bsnes/nall/dl.hpp
+++ b/bsnes/nall/dl.hpp
@@ -37,6 +37,7 @@
     if(handle) close();
     handle = (uintptr_t)dlopen(string(path, *path && !strend(path, "/") ? "/" : "", "lib", name, ".so"), RTLD_LAZY);
     if(!handle) handle = (uintptr_t)dlopen(string("/usr/local/lib/lib", name, ".so"), RTLD_LAZY);
+    if(!handle) handle = (uintptr_t)dlopen(string("/usr/lib/bsnes/lib", name, ".so"), RTLD_LAZY);
     return handle;
   }
 
